packages/fast-components-react-msft/src/button/button.tsx

```diff
@@ -37,7 +37,7 @@ class Button extends Foundation<IButtonHandledProps & IManagedClasses<IMSFTButto
      */
     protected generateClassNames(): string {
         if (!this.props.appearance) {
-            return;
+            return super.generateClassNames();
```

Comment:
Super still gets added by the base button, so I don't think this is necessary? If it is, I would like to take care of it in my PR since I am altering button quite a bit.